src/AuthorizedSocketConnection.js

```diff
@@ -79,6 +85,8 @@ export default class AuthorizedSocketConnection<TContext, TCredentials> {
     cb?: Function,
   ) => {
     if (this.subscriptions.has(id)) {
+      this.log('debug', `Duplicate subscription attempted`, { id });
```

Comment:
don't use template strings when we don't have templates?

Reply:
do you prefer `this.log('debug', ...)` over `this.logger.debug(...)`? is that just more like the winston API?